samples/dashboards/demos/climate/demo.js

```diff
@@ -1,50 +1,159 @@
 /* eslint-disable prefer-const, jsdoc/require-description */
-const { Range: RangeModifier } = Dashboards.DataModifier.types;
 
-const dataPool = new Dashboards.DataPool();
-const dataScopes = {
-    FD: 'Days with Frost',
-    ID: 'Days with Ice',
-    RR: 'Days with Rain',
-    TN: 'Average Temperature',
-    TX: 'Maximal Temperature'
-};
-const dataTemperatures = {
-    C: 'Celsius',
-    F: 'Farenheit',
-    K: 'Kelvin'
-};
-const initialMin = Date.UTC(2010, 0, 5);
-const minRange = 30 * 24 * 3600 * 1000; // 30 days
-const maxRange = 2 * 365 * 24 * 3600 * 1000; // 2 years
-const defaultCity = 'New York';
-const defaultData = 'TXC';
-
-let citiesData;
-let citiesMap;
-let citiesTable;
-let cityGrid;
-let cityScope = defaultCity;
-let citySeries;
-let dataScope = defaultData;
-let navigatorSeries;
-let worldDate = Date.UTC(2010, 11, 25); // months start with 0
-let kpis = {};
-let darkMode = false;
-let temperatureScale = 'C';
-
-async function setupDashboard() {
-
-    citiesData = await buildCitiesData();
-    buildSymbols();
-
-    const cityConnector = await dataPool.getConnector(cityScope);
-    const map = await fetch(
-        'https://code.highcharts.com/mapdata/custom/world.topo.json'
-    ).then(response => response.json());
-    const mapPoints = await buildCitiesMap();
-
-    return new Promise(resolve => Dashboards.board('container', {
+// left arrow
+Highcharts.SVGRenderer.prototype.symbols.leftarrow = (x, y, w, h) => [
+    'M', x + w / 2 - 1, y,
+    'L', x + w / 2 - 1, y + h,
+    x - w / 2 - 1, y + h / 2,
+    'Z'
+];
+// right arrow
+Highcharts.SVGRenderer.prototype.symbols.rightarrow = (x, y, w, h) => [
+    'M', x + w / 2 + 1, y,
+    'L', x + w / 2 + 1, y + h,
+    x + w + w / 2 + 1, y + h / 2,
+    'Z'
+];
+
+const MathModifier = Highcharts.DataModifier.types.Math; // @todo switch module
+const RangeModifier = Dashboards.DataModifier.types.Range;
+
+const colorStopsDays = [
+    [0.0, '#C2CAEB'],
+    [1.0, '#162870']
+];
+const colorStopsTemperature = [
+    [0.0, '#4CAFFE'],
+    [0.3, '#53BB6C'],
+    [0.5, '#DDCE16'],
+    [0.6, '#DF7642'],
+    [0.7, '#DD2323']
+];
+
+setupBoard();
+
+async function setupBoard() {
+    let activeCity = 'New York',
+        activeColumn = 'TX',
```

Comment:
Why change to max temp on the first render?

Reply:
It is the default because it is colorful.